Dockerfile

```diff
@@ -1,22 +1,3 @@
-FROM registry.centos.org/centos/centos:7
-
-MAINTAINER Mohammed Zeeshan Ahmed <moahmed@redhat.com>
-
-#RUN yum -y update && yum clean all
-
-RUN mkdir -p /opt/scripts /var/www/html
-
-ADD dist /var/www/html/
-
-ADD ./fix-permissions.sh ./install.sh ./passwd.template ./run.sh /opt/scripts/
-
-RUN chmod -R 777 /opt/scripts && . /opt/scripts/install.sh
-
-WORKDIR /var/www/html
-
-EXPOSE 8080 8443
-
-USER apache
-
-ENTRYPOINT ["/opt/scripts/run.sh"]
-CMD ["apache"]
+# Builds a Docker to deliver dist/
+FROM nginx:latest
```

Comment:
This seems to be Debian based docker image. I had an impression that we are using centos based images for fabric8